yarn.lock

```diff
@@ -5510,6 +5510,11 @@ jsbn@~0.1.0:
   version "0.1.1"
   resolved "https://registry.yarnpkg.com/jsbn/-/jsbn-0.1.1.tgz#a5e654c2e5a2deb5f201d96cefbca80c0ef2f513"
 
+jschardet@1.6.0:
```

Comment:
@theia-ide/ip-help it is not MIT, can we use it?

Reply:
Normally we would not be able to use/distribute such a `LGPL-licensed` dependency, but in this specific instance, we already requested a special permission for `jschardet v1.6.0`, that was granted by the Eclipse Foundation Board. So I think it's ok.  

See this CQ for more details: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20251

Reply:
Hey, I wonder how can I view this website: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20251? I tried to log in with my eclipse account, but it shows "The username or password you entered is not valid." Is it that only committer of Eclipse can use ipzilla?

Reply:
@a1994846931931 Do you have an eclipse account? @theia-ide/ip-help is it true that only official Eclipse Theia Committer can access it?

Reply:
@akosyakov I have an eclipse account, but I am only a contributor, not a committer.

Reply:
> @a1994846931931 Do you have an eclipse account? @theia-ide/ip-help is it true that only official Eclipse Theia Committer can access it?

Yes, IIRC this is correct. Committers are expected to take care of CQs.